cfpb/consumerfinance.gov
# Update Paying for College Helper Text Size

## Changes

- Changed font size from 5px to 16px to match the Design System standard

## How to test this PR

1. Run locally
2. [Go to Paying for College tool](http://127.0.0.1:8000/paying-for-college/your-financial-path-to-graduation/?iped=176080&pid=1408-3&houp=onCampus&typp=bachelors&prop=0&lenp=4&ratp=inState&depp=dependent&cobs=n&utm_source=vanity&utm_medium=redirect&utm_campaign=gradpath&tuit=8650&hous=9815&book=1200&subl=2500&unsl=2000&houx=1265&fdx=537&clhx=131&trnx=858&hltx=328&entx=193&retx=393&taxx=344&othx=370)
3. Go through steps and notice text size change 


## Screenshots

Before:
<img width="598" alt="image" src="https://github.com/cfpb/consumerfinance.gov/assets/85513449/b838b471-f8f3-4ab0-8e05-687b8954c740">

After:
<img width="609" alt="image" src="https://github.com/cfpb/consumerfinance.gov/assets/85513449/22832c10-ae24-4156-a9ab-c06ea5a7ece7">

## Notes and todos

- This was the quickest/simplest implementation however it's not exactly how the Design System implements helper text. For some reason, this helper text was put into a Jinja macro and used custom CSS. I didn't want to mess with the implementation in the event it had to be this way for reasons unbeknownst to me.
- Resolves XP issue 144
